baxter_collaboration_lib/src/robot_utils/ros_thread_image.cpp

```diff
@@ -4,10 +4,11 @@
 /*                          ROSThreadImage                                */
 /**************************************************************************/
 
-ROSThreadImage::ROSThreadImage(std::string _name) :  nh(_name), name(_name), is_closing(false),
+ROSThreadImage::ROSThreadImage(std::string _name, std::string encoding) :  nh(_name), name(_name), is_closing(false),
```

Comment:
indentation: move ` nh(_name), name(_name), is_closing(false),` to a new line (and re-arragne the things afterwards accordingly)